.ci/atime/tests.R

```diff
@@ -40,6 +40,7 @@ for (extra.arg in extra.args.6107){
 # @note Please check https://github.com/tdhock/atime/blob/main/vignettes/data.table.Rmd for more information.
 # nolint start: undesirable_operator_linter. ':::' needed+appropriate here.
 test.list <- atime::atime_test_list(
+  N = as.integer(10^seq(1, 7, by=0.25)),
```

Comment:
WDYT about naming like `N_grid` or `commonN` to make it more obvious that this is a constant defined & re-used here? I sorry with just plain `N` that it might take a bit longer to catch on to that.

Reply:
I'll add a comment